notebook/notebookapp.py

```diff
@@ -398,12 +398,12 @@ class NbserverListApp(JupyterApp):
                   "details from the server info file."))
 
     def start(self):
-        if not self.json:
-            print(_("Currently running servers:"))
-        for serverinfo in list_running_servers(self.runtime_dir):
-            if self.json:
-                print(json.dumps(serverinfo))
-            else:
+        serverinfo_list = list(list_running_servers(self.runtime_dir))
+        if self.json:
+            print(json.dumps(serverinfo_list))
```

Comment:
Maybe pass `indent=2` so that the JSON is nicely formatted if a human needs to look at it?